checkov/helm/runner.py

```diff
@@ -126,16 +126,34 @@ def run(self, root_folder, external_checks_dir=None, files=None, runner_filter=R
 
                 self.parse_helm_dependency_output(o)
 
-                try:
-                    # --dependency-update needed to pull in deps before templating.
-                    proc = subprocess.Popen([self.helm_command, 'template', '--dependency-update', chart_dir], stdout=subprocess.PIPE, stderr=subprocess.PIPE)  # nosec
-                    o, e = proc.communicate()
-                    logging.debug(
-                        f"Ran helm command to template chart output. Chart: {chart_meta['name']}. dir: {target_dir}. Output: {str(o, 'utf-8')}")
+                if runner_filter.var_files != None:
```

Comment:
```suggestion
                if runner_filter.var_files:
```
this way you also skip this part, when var_files is an empty list